description.md

```diff
@@ -18,8 +17,34 @@ Each topic in this file contains a small description and links to the files cove
 
 ### Assignments
 
-### Drills
+Assignments are practical activities that learners undertake individually or in teams over the span of multiple weeks.
+The primary objective of an assignment is to create a compelling scenario that allows learners to apply the concepts they have learned.
+
+#### Support
```

Comment:
I think the title is misleading. This might better refer to mentorship or guidance. Support is either material or somebody telling you what to do, think manuals and technical support for a software solution